### PR TITLE
Fix version padding

### PR DESCRIPTION
**Pull request by wli3**

### CODE REVIEW

File: build.proj [hunk 1]
```diff
@@ -23,14 +23,21 @@
              Directories="obj" />
     <Exec Command="git rev-list --count HEAD" 
           ConsoleToMSBuild="true">
-      <Output TaskParameter="ConsoleOutput" PropertyName="CommitCount" />
+      <Output TaskParameter="ConsoleOutput" PropertyName="GitInfoCommitCount" />
     </Exec>
 
+    <ItemGroup>
+      <GitInfoCommitCountLines Include="$(GitInfoCommitCount)" />
+    </ItemGroup>
```
Comment (nguerrera) [inline build.proj]: Why is this necessary?

File: build.proj [hunk 2]
```diff
@@ -23,14 +23,21 @@
...
+
     <PropertyGroup>
+      <GitInfoCommitCount>%(GitInfoCommitCountLines.Identity)</GitInfoCommitCount>
```
Comment (nguerrera) [inline build.proj]: Ditto.

File: build.proj (hunk 1 above)
Comment (wli3) [inline build.proj]: I had the same question, I used only one variable without the extra itemgroup. It just won't work. I get 00000 as result. After 3 round i ended up adding all the ceremonies in CLI repo.

Comment (wli3) [inline build.proj]: At this point I'm sure it is the minimal ceremony. But I will at around

**Comment by livarcocc** [comment]
@wli3 talk to @nguerrera tomorrow to replace this with a timestamp based version.

**Comment by nguerrera** [comment]
> @wli3 talk to @nguerrera tomorrow to replace this with a timestamp based version.

I don't have a problem with commit count versioning. I was just curious about how that msbuild code works, but if we have it in other repos, it's a separate issue to clean it up.
